src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2856,9 +2875,9 @@ namespace UnitarySystems {
                 }
 
                 Real64 loc_m_NoCoolHeatSAFMethod_FlowPerHeatingCapacity(-999.0);
-                if (fields.find("heating_supply_air_flow_rate_per_unit_of_capacity_during_heating_operation") != fields.end()) { // not required field
+                if (fields.find("no_load_supply_air_flow_rate_per_unit_of_capacity_during_heating_operation") != fields.end()) { // not required field
```

Comment:
Replaced "heating" with "no_load" to match field name to the idd, and expected by the json input processor.